packages/google_maps_flutter/example/lib/padding.dart

```diff
@@ -0,0 +1,81 @@
+import 'dart:math';
+import 'package:flutter/material.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+import 'page.dart';
+
+class PaddingPage extends Page {
+  PaddingPage() : super(const Icon(Icons.map), 'Add padding to the map');
+
+  @override
+  Widget build(BuildContext context) {
+    return const MarkerIconsBody();
+  }
+}
+
+class MarkerIconsBody extends StatefulWidget {
+  const MarkerIconsBody();
+
+  @override
+  State<StatefulWidget> createState() => MarkerIconsBodyState();
+}
+
+const LatLng _kMapCenter = LatLng(52.4478, -3.5402);
+
+class MarkerIconsBodyState extends State<MarkerIconsBody> {
+  GoogleMapController controller;
+
+  double _padding = 0;
+
+  @override
+  Widget build(BuildContext context) {
+    final GoogleMap googleMap = GoogleMap(
+      onMapCreated: _onMapCreated,
+      initialCameraPosition: const CameraPosition(
+        target: _kMapCenter,
+        zoom: 7.0,
+      ),
+      padding: EdgeInsets.only(bottom: _padding),
+    );
+
+    final List<Widget> columnChildren = <Widget>[
+      Padding(
+        padding: const EdgeInsets.all(10.0),
+        child: Center(
+          child: SizedBox(
+            width: 300.0,
+            height: 200.0,
+            child: googleMap,
+          ),
+        ),
+      ),
+    ];
+
+    columnChildren.add(_setPaddingButton());
+
+    return Column(
+      mainAxisAlignment: MainAxisAlignment.start,
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      children: columnChildren,
+    );
+  }
+
+  void _onMapCreated(GoogleMapController controllerParam) {
+    setState(() {
+      controller = controllerParam;
+    });
+  }
+
+  Widget _setPaddingButton() {
+    return Padding(
+      padding: const EdgeInsets.all(8.0),
+      child: RaisedButton(
+        child: const Text('Change padding'),
+        onPressed: () async {
+          setState(() {
+            _padding = Random().nextInt(150).toDouble();
```

Comment:
Can you take some kind of a user input here? Perhaps slider with some padding scale? I think that will illustrate this much better.